cmd/kubeadm/app/util/runtime/runtime.go

```diff
@@ -197,7 +189,7 @@ func detectCRISocketImpl(isSocket func(string) bool) (string, error) {
 	foundCRISockets := []string{}
 	knownCRISockets := []string{
 		// Docker and containerd sockets are special cased below, hence not to be included here
-		"/var/run/crio/crio.sock",
+		"unix:///var/run/crio/crio.sock",
```

Comment:
i think this may actually fail because we use os.Stat for checking file existence:
https://github.com/kubernetes/kubernetes/blob/c1e69551be1a72f0f8db6778f20658199d3a686d/cmd/kubeadm/app/util/runtime/runtime_unix.go#L32

we probably have to keep constants with file paths and prepending unix:// only when passing the flag to the kubelet.